pyccel/codegen/printing/ccode.py

```diff
@@ -250,18 +254,38 @@ def _print_Complex(self, expr):
                         PyccelMul(expr.imag, ImaginaryUnit()))))
 
     def _print_PythonComplex(self, expr):
-        self._additional_imports.add("complex.h")
+        self._additional_imports.add("complex")
         return self._print(PyccelAssociativeParenthesis(PyccelAdd(expr.real_part,
                         PyccelMul(expr.imag_part, ImaginaryUnit()))))
 
     def _print_ImaginaryUnit(self, expr):
         return '_Complex_I'
 
+    def _print_ModuleHeader(self, expr):
+        name = expr.module.name
+        # TODO: Add classes and interfaces
+        funcs = '\n\n'.join('{};'.format(self.function_signature(f)) for f in expr.module.funcs)
+
+        # Print imports last to be sure that all additional_imports have been collected
+        imports = [*expr.module.imports, *map(Import, self._additional_imports)]
+        imports = '\n'.join(self._print(i) for i in imports)
+
+        return ('#ifndef {name}_H\n'
+                '#define {name}_H\n\n'
+                '{imports}\n\n'
+                #'{classes}\n\n'
+                '{funcs}\n\n'
+                #'{interfaces}\n\n'
+                '#endif // {name}_H').format(
+                        name    = name,
```

Comment:
use `name,.upper()`.
I will approve this PR when you resolve this.

Reply:
I wondered about this. I didn't use `name.upper()` as I was worried about name clashes if we have a `test.py` and a `Test.py` file. But it's probably not worth ignoring the convention just in case the user has bad names. This won't work in fortran anyway as fortran is not case sensitive.
I have changed to `name.upper()`